.github/workflows/ci.yml

```diff
@@ -0,0 +1,46 @@
+name: CI
+
+on: 
+  pull_request:
+    branches:
+      - dev
+      - main
+      - 'feature/*'
+  push:
+    branches:
+      - dev
+      - main
+      - 'feature/*'
+
+jobs:
+  ci-job:
+    runs-on: ubuntu-latest
+    steps:
+    - name: Checkout
+      uses: actions/checkout@v2
+    - name: Set up NodeJS
+      uses: actions/setup-node@v3
+      with:
+        node-version: 16.0
```

Comment:
I assume we are using node version `16.0` for development as well?

Reply:
For development, i am using `16.15.1` but anything `>= 16` should be fine.